packages/did/src/DidLinks/AccountLinks.chain.ts

```diff
@@ -163,17 +163,12 @@ export function connectedAccountsFromChain(
  * @param linkedAccount The account to use for the lookup.
  * @returns The Web3 name linked to the given account, or `null` otherwise.
  */
-export async function queryWeb3Name(
-  linkedAccount: Address
-): Promise<Web3Name | null> {
+export async function queryWeb3Name(linkedAccount: Address): Promise<Web3Name> {
```

Comment:
I feel like `queries` should still return null, if nothing could be queried,

Reply:
Renamed all query to fetch :)